config/environments/development.rb

```diff
@@ -44,7 +44,7 @@
     domain:                'gmail.com',
     authentication:        'plain',
     enable_startttls_auto: true,
-    user_name:             ENV['EMAIL_USER'],
-    password:              ENV['EMAIL_PASS']
+    user_name:             ENV['gmail_username'],
```

Comment:
@gitbnw how would the user set the environment variable? .. I used to launch before this by saying $EMAIL_USER='user@example.com' EMAIL_PASS='password' rails s .. after this figaro gem, how would this work? Thanks!